spanner/google/cloud/spanner_v1/client.py

```diff
@@ -51,6 +61,11 @@
 )
 
 
+if os.getenv('SPANNER_EMULATOR_HOST') != None and \
+    ('http://' in os.getenv('SPANNER_EMULATOR_HOST') or 'https://' in os.getenv('SPANNER_EMULATOR_HOST')):
```

Comment:
Worth an abstraction like get_spanner_emulator_host() as opposed to bare getenv calls?

Reply:
Done.